spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/actions/RewriteDataFilesSparkAction.java

```diff
@@ -159,7 +159,7 @@ public RewriteDataFiles.Result execute() {
 
     validateAndInitOptions();
 
-    Map<StructLike, List<List<FileScanTask>>> fileGroupsByPartition =
```

Comment:
Can we also add `EMPTY_RESULT` static variable like in the action for position deletes?

Reply:
ok. Added.